src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToTimestampTransformer.php

```diff
@@ -29,20 +29,19 @@ class DateTimeToTimestampTransformer extends BaseDateTimeTransformer
      * @return int A timestamp
      *
      * @throws TransformationFailedException If the given value is not an instance
-     *                                       of \DateTime or if the output
-     *                                       timezone is not supported.
+     *                                       of \DateTime.
      */
-    public function transform($value)
+    public function transform($dateTime)
```

Comment:
-1 for this renaming. It will make it harder to merge branches together for no real benefit